src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
+  }
+  if (operation === '%') {
+    return one.mod(two).toString();
+  }
+  throw Error(`Unknown operation '${operation}'`);
+}
```

Comment:
- [ ] Kindly, display the user friendly message to the users once he or she performs incorrect or invalid number

![calcc4](https://user-images.githubusercontent.com/55635977/133627763-bf827232-f797-4919-9cef-15781829d8b1.png)

Reply:
Thanks, I'll implement that.